conf/messages

```diff
@@ -54,6 +54,7 @@ vatDetails.checkWhatYouOwe = Check what you owe and make a payment
 returnObligation.noReturn = No returns due right now
 returnObligation.nextReturn = Next return due
 returnObligation.viewReturns = View return deadlines
+returnObligation.submit = Submit return
```

Comment:
Do we need the Welsh Translation for this? Or will this be covered in another story?